doc/neps/Makefile

```diff
@@ -1,5 +1,6 @@
 # Minimal makefile for Sphinx documentation
 #
+PYVER = 3.6
```

Comment:
since the python code uses f-strings, we need at least python3.6 I copied the solution used in doc/Makefile, so any adjustment here should be done there as well

Reply:
hmm, it seems that pinning breaks the build